doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1671 @@
+  - fdb_entry_oid_H2 pointing to bridgeport_oid_1 and fdb_entry_oid_H3 pointing to bridgeport_oid_2.
+
+#### 3.4.3.2 BUM handling
+- Refer to Sec 1.2.1.1
+- DF settings
+  - At VTEP1 bridgeport_oid for Po2 is marked as NON_DF.
+  - At VTEP4 bridgeport_oid for Po1 is marked as NON_DF.
+  - At VTEP2, VTEP3 bridgeport_oid for Po2 is marked as NON_DF.
+- Isolation Group settings for Split Horizon and Local Bias
+  - At VTEP1
+    - tnl_oid_2-4 for tunnels towards the peer multihoming VTEP2-4
+    - bridgeport_oid_2-4 corresponding to tunnels as above.
+    - isogrp_oid_2-4 corresponding to tunnels as above.
+    - bridgeport_oid_2-4 have isolation group attribute set as isogrp_oid_2-4.
+    - isogrp_mbr_oid_21 corresponding to isogrp_oid_2+bridgeport_oid_po2
+    - isogrp_mbr_oid_31 corresponding to isogrp_oid_3+bridgeport_oid_po2
+    - isogrp_mbr_oid_41 corresponding to isogrp_oid_4+bridgeport_oid_po1
+    - isogrp_mbr_oid_42 corresponding to isogrp_oid_4+bridgeport_oid_po2
+  - At VTEP4
+    - tnl_oid_1-3 for tunnels towards the peer multihoming VTEP1-3
+    - bridgeport_oid_1-3 corresponding to tunnels as above.
+    - isogrp_oid_1-3 corresponding to tunnels as above.
+    - bridgeport_oid_1-3 have isolation group attribute set as isogrp_oid_1-3.
+    - isogrp_mbr_oid_21 corresponding to isogrp_oid_2+bridgeport_oid_po2
+    - isogrp_mbr_oid_31 corresponding to isogrp_oid_3+bridgeport_oid_po2
+    - isogrp_mbr_oid_11 corresponding to isogrp_oid_1+bridgeport_oid_po1
+    - isogrp_mbr_oid_12 corresponding to isogrp_oid_1+bridgeport_oid_po2
+  - At VTEP2
+    - tnl_oid_1,3,4 for tunnels towards the peer multihoming VTEP1,3,4
+    - bridgeport_oid_1,3,4 corresponding to tunnels as above.
+    - isogrp_oid_1,3,4 corresponding to tunnels as above.
+    - bridgeport_oid_1,3,4 have isolation group attribute set as isogrp_oid_1,3,4.
+    - isogrp_mbr_oid_11 corresponding to isogrp_oid_1+bridgeport_oid_po2
+    - isogrp_mbr_oid_31 corresponding to isogrp_oid_3+bridgeport_oid_po2
+    - isogrp_mbr_oid_41 corresponding to isogrp_oid_4+bridgeport_oid_po2
+  - At VTEP3
+    - tnl_oid_1,2,4 for tunnels towards the peer multihoming VTEP1,2,4
+    - bridgeport_oid_1,2,4 corresponding to tunnels as above.
+    - isogrp_oid_1,2,4 corresponding to tunnels as above.
+    - bridgeport_oid_1,2,4 have isolation group attribute set as isogrp_oid_1,2,4
+    - isogrp_mbr_oid_11 corresponding to isogrp_oid_1+bridgeport_oid_po2
+    - isogrp_mbr_oid_21 corresponding to isogrp_oid_2+bridgeport_oid_po2
+    - isogrp_mbr_oid_41 corresponding to isogrp_oid_4+bridgeport_oid_po2
+
+<a id="35-CLI"></a>
+## 3.5 CLI
+
+<a id="351-Configuration-Commands"></a>
+### 3.5.1 Configuration Commands
+
+**Ethernet-segment config commands**
+```
+sudo config interface <interface name> sys-mac XX:XX:XX:XX:XX:XX
+sudo config interface PortChannel1 evpn-esi {XX:XX:XX:XX:XX:XX:XX:XX:XX:XX | auto-lacp | auto-system-mac}
+sudo config interface PortChannel1 evpn-df-pref (1-65535)
```

Comment:
- These commands do not follow the "config interface" command syntax (https://github.com/sonic-net/sonic-utilities/blob/0e6a55ef5eac306ef61d6f0241625a6baee42ab8/doc/Command-Reference.md?plain=1#L5227)
- "sudo config interface PortChannel1 evpn-esi {XX:XX:XX:XX:XX:XX:XX:XX:XX:XX | auto-lacp | auto-system-mac}" command does not allow to configure a local discriminator for type-3 ESI. Also, it looks like there is no type-1 (auto-lacp) ESI support in FRR. (NOTE: this comment is connected to the one with EVPN_ETHERNET_SEGMENT table)

**Proposed commands:**
sudo config interface sys-mac <interface_name> XX:XX:XX:XX:XX:XX
sudo config interface evpn-esi <interface_name> {type-0 XX:XX:XX:XX:XX:XX:XX:XX:XX:XX | type-3 (1-16777215)}
sudo config interface evpn-df-pref <interface_name> (1-65535)

Reply:
thank you. Updated